Northstar.Custom/mod/scripts/vscripts/sh_custom_air_accel.gnut

```diff
@@ -8,7 +8,9 @@ void function CustomAirAccelVars_Init()
 	
 	#if SERVER
 		AddCallback_OnPlayerRespawned( ApplyCustomPlayerAirAccel )
-		AddCallback_OnPilotBecomesTitan( ApplyCustomPlayerAirAccelFromTitan ) // not sure if necessary but assuming it is
+		AddCallback_OnTitanBecomesPilot( ApplyCustomPlayerAirAccelFromTitan ) // airaccel is reset after player leaves titan
+		AddCallback_OnPilotBecomesTitan( ApplyCustomPlayerAirAccelFromTitan ) // airaccel is also reset after player enters titan
+		AddCallback_OnPlayerGetsNewPilotLoadout( ApplyCustomPlayerAirAccelOnLoadoutChange ) // airaccel is also reset on loadout change for some reason
```

Comment:
This line breaks the game for me, I can't even get to the Northstar menu. 
![image](https://user-images.githubusercontent.com/16687318/147889642-c98e8396-ac3d-4927-b86e-f57d9e8e190a.png)

Reply:
Forgot to add the function - Will fix now.